README.md

```diff
@@ -25,7 +25,15 @@ This is a standard autoconf package. So:
     make && make install
 
 It is likely that you will need to change system-dependent paths such as the
-xorg module directory. See `configure --help` for options.
+xorg module directory. Otherwise mtrack may be not installed in xserver search
```

Comment:
Thanks for the additional comment on the xorg module directory. However, I would like to keep distro specific packaging information out of the README. Please remove it.